src/main/scala/org/ergoplatform/api/WalletApiRoute.scala

```diff
@@ -0,0 +1,67 @@
+package org.ergoplatform.api
+
+import akka.actor.{ActorRef, ActorRefFactory}
+import akka.http.scaladsl.server.{Directives, Route}
+import akka.pattern.ask
+import org.ergoplatform.modifiers.mempool.ErgoTransaction
+import org.ergoplatform.nodeView.ErgoReadersHolder.{GetReaders, Readers}
+import org.ergoplatform.nodeView.wallet.{ErgoWalletReader, PaymentRequest, PaymentRequestDecoder}
+import org.ergoplatform.settings.ErgoSettings
+import scorex.core.NodeViewHolder.ReceivableMessages.LocallyGeneratedTransaction
+import scorex.core.api.http.ApiError.BadRequest
+import scorex.core.api.http.ApiResponse
+import scorex.core.settings.RESTApiSettings
+
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+
+case class WalletApiRoute(readersHolder: ActorRef, nodeViewActorRef: ActorRef, ergoSettings: ErgoSettings)
```

Comment:
Looks like there is no way to get addresses of out wallet. Let's add a method that return sequence of addresses of our wallet, otherwise it is quite useless as soon as we do not know, how to request money

Reply:
introduced `wallet/addresses` API method